python/ray/experimental/worker.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray
+import numpy as np
+
+
+def get(object_ids, worker=None):
+    """Get a remote object or a collection of remote objects from the object
+    store.
+
```

Comment:
Could you add a note explaining how this differs from `ray.get`. The block below is probably not necessary, since you can assume people know how `ray.get` works.